src/Exsurge.Gabc.js

```diff
@@ -993,6 +995,14 @@ export var Gabc = {
         // rhythmic markings
         case '.':
           mark = new Markings.Mora(note, ctxt.staffInterval / 4.0);
+          for(j = notes.length - 1; j >= 0 && data[++i] === '.'; --j) {
+            var earlierNote = notes[j];
```

Comment:
Double declaration of `earlierNote` and `earlierMark`. Scope of variable is a function, not a switch block.